runtime/compiler/x/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -199,7 +199,7 @@ uint8_t *J9::X86::AheadOfTimeCompile::initializeAOTRelocationHeader(TR::Iterated
          cursor += SIZEPOINTER;
 
          TR_OpaqueClassBlock *j9class = (TR_OpaqueClassBlock *) aconstNode->getAddress();
-         uintptrj_t classChainOffsetInSharedCache = sharedCache->lookupClassChainOffsetInSharedCacheFromClass(j9class);
+         uintptrj_t classChainOffsetInSharedCache = sharedCache->getClassChainOffsetOfIdentifyingLoaderForClazzInSharedCache(j9class);
```

Comment:
Why the change in name from `lookupClassChainOffsetInSharedCacheFromClass` to `getClassChainOffsetOfIdentifyingLoaderForClazzInSharedCache`

Reply:
`lookupClassChainOffsetInSharedCacheFromClass` was not present in mainline.
When I added it, `getClassChainOffsetOfIdentifyingLoaderForClazzInSharedCache` is the name Irwin and I agreed on.